gulp/tasks/test_ci.js

```diff
@@ -0,0 +1,14 @@
+(function () {
```

Comment:
Yes; I've added a `--ci` option to enable testing with multiple browsers. It's enabled in `.travis.yml`.